mobile/library/cc/engine_builder.cc

```diff
@@ -938,6 +953,12 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
     ads_config->set_api_type(api_type_enum);
     ads_config->add_grpc_services()->mutable_google_grpc()->set_target_uri(target_uri);
   }
+  if (enable_cds_) {
+    auto* cds_config = bootstrap->mutable_dynamic_resources()->mutable_cds_config();
```

Comment:
We'll also need to support xdstp wildcard, once https://github.com/envoyproxy/envoy/pull/25115 lands (which should be today), which means we'd have to set https://github.com/envoyproxy/envoy/blob/main/test/integration/ads_integration_test.cc#L1689.  If you're ready to submit this PR before the other PR lands, if you can please add a TODO to add xdstp support and come back to it.  Thanks!